src/behavior-considered-undefined.md

```diff
@@ -1,9 +1,14 @@
 ## Behavior considered undefined
 
-Rust code, including within `unsafe` blocks and `unsafe` functions is incorrect
-if it exhibits any of the behaviors in the following list. It is the
-programmer's responsibility when writing `unsafe` code that it is not possible
-to let `safe` code exhibit these behaviors.
+Rust code is incorrect if it exhibits any of the behaviors in the following
+list. This includes code within `unsafe` blocks and `unsafe` functions!
+`unsafe` is not an excuse to cause undefined behavior.
```

Comment:
```suggestion
Use of `unsafe` should never result in undefined behavior.
```

Reply:
"should" is way too weak IMO. If you don't like "excuse", what about

> Even with `unsafe`, Rust code must never cause undefined behavior.

I also considered something like

> `unsafe` only means that avoiding undefined behavior is on the programmer; it does not change anything about the fact that Rust programs must never cause undefined behavior.

Reply:
I like the second version  better although "must" is definitely better than "should", I agree.